### ISSUE TITLE
Release on rubygems.org

### ISSUE THREAD

**Issue by postmodern** [digininja/CeWL#102]
It would make it easier for users to install CeWL if it were available on https://rubygems.org.

**Comment by digininja** [comment]
Is it easy to put it there? Do you have instructions?

**Comment by postmodern** [comment]
Yes, there are [guides](https://guides.rubygems.org/make-your-own-gem/) for how to publish your first gem. You just need a rubygems.org account, create a `cewl.gemspec` file (`bundle gem` can do this), fill in your gem's metadata/dependencies, build the `.gem`, and `gem push` the `.gem` file. You can automate the release process by adding `require "bundler/gem_tasks"` to your `Rakefile` or use [rubygems-tasks](https://github.com/postmodern/rubygems-tasks#readme).
